masterlist.yaml

```diff
@@ -13180,7 +13187,27 @@ plugins:
     req: [ 'Better descriptions SSE.esp' ]
   - name: 'Better Descriptions SSE - YOT Patch.esp'
     req: [ 'YOT - Your Own Thoughts.esp' ]
-
+  - name: 'RealisticWaterTwo - Unique Region Names.esp'
```

Comment:
```suggestion

  - name: 'RealisticWaterTwo - Unique Region Names.esp'
```

Reply:
Now I see the logic, I had assumed the existing blank line was a typo.